bitnami/grafana/templates/ingress.yaml

```diff
@@ -17,12 +17,18 @@ spec:
     - host: {{ .name | quote }}
       http:
         paths:
+        {{- range .extraPaths }}
+        - path: {{ .path }}
+          backend:
+            serviceName: {{ .backend.serviceName }}
+            servicePort: {{ .backend.servicePort}}
+        {{- end }}
         {{- $paths := default (list "/") .paths }}
         {{- range $paths }}
-          - path: {{ . }}
-            backend:
-              serviceName: {{ include "grafana.fullname" $ }}
-              servicePort: http
+        - path: {{ . }}
+          backend:
+            serviceName: {{ include "grafana.fullname" $ }}
+            servicePort: http
```

Comment:
Please indent this array. 
```yaml
array:
  - item1
  - item2
```

Reply:
Hi @dani8art 

I create commit. Thanks.. :-)